lib/src/models/item_model.dart

```diff
@@ -0,0 +1,22 @@
+import 'result_model.dart';
+
+class ItemModel {
+  int _page;
+  List<Result> _results = [];
+
+  ItemModel.fromJson(Map<String, dynamic> parsedJson) {
+    print(parsedJson['results'].length);
```

Comment:
Just as a good practice, dont leave prints on the code, if you really want to log this at a development level lets try and use this:

`assert(() { debugPrint(parsedJson['results'].length) }());`

Reply:
Done, i removed the print.